CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+We welcome contributions via both the raising of issues and submitting pull requests. But before you
+do, please take a moment to consult the below.
```

Comment:
Consider rewording - "please take a moment to consult the below." > "please take a moment to review the following."

Reply:
done